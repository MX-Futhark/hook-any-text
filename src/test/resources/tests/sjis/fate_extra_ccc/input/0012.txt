82A882E2814282A082A282A982ED82E782B8836C836283678FE382C50A96B3916F82B582C482A282E982CC82C582B782CB81412343323336313936303033835683938357234344454682CD81420A815C815C815C82D381420000000000000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C0000001D0000001E0000001F00000020000000210000002200000B000000C014C9080000000000000000000000000000580000000000FFFFFFDC0000F043FFFFFFFF00000000000000000000A842FFFFFFDC00000000FFFFFFFF00005043000000000000A842FFFFFFDC0000F043FFFFFFFF00000000000088430B0000000000000000000000FFFFFFFF000050430000884300000000000000000B0000004015C9080000000000000000000000000200580000000000FFFFFFDC00000A43FFFFFFFF000000000000000000001843FFFFFFDC000060C1FFFFFFFF000050430000000000001843FFFFFFDC00000A43FFFFFFFF00000000000088430B000000A015C90800000000FFFFFFFF0000504300008843000100000000000002000000000000000000000000000000000000000C00C20100000000E42EA3480B0000006017C90800000000000000000000000002001000000000000000000000000000FFFFFFDC0000000000003C43000000000000F04300000000FFFFFFDC0000F04300003C4300000000000000000000A842FFFFFFDC0000000000008843000000000000F0430000A842FFFFFFDC0000F0430000884300000000FFFFFFDC0B0000000000000000000000000000000000000003001000000000000000000000000000FFFFFFDC000060C100001F43000000000000184300000000FFFFFFDC00000A4300001F43000000000000000000001843FFFFFFDC000060C100809B430000000000001843