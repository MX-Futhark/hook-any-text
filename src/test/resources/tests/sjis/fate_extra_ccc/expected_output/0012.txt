["おや。あいかわらずネット上で
無双しているのですね、シンジは。
———ふ。"]